### PR TITLE
Lorem text + placeholder for img. Formulated CSS global vs CSS pages.

### PR DESCRIPTION
**Pull request by minna-le**
Created new branch named "Landing page-myStory section".
Inserted Lorem text to myStory section's text.
Inserted placeholder to myStory section's profile picture.
Formulated global CSS and CSS files for pages (index.css for homepage, myStory.css for myStory page)
Finally, Pushed these new changes after Trunio's review to this new branch and asked for review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Happy Hearts Roaming</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="styles/index.css" />
```
Comment (Truniole) [inline index.html:7]: Great 👏

File: index.html [hunk 2]
```diff
@@ -41,20 +42,24 @@ <h1>Happy Hearts Roaming</h1>
...
           <div>
             <p>
-              I am Minna, an aspiring software developer from Finland and a
-              proud mom of a spirited 3-year-old. I feel fortunate to have my
-              parents, who are still healthy, from Vietnam, along with my
-              beloved sister from the US, who is not only very close to me but
-              also shares my passion for traveling together. Our family
-              frequently travels across continents to reconnect and create
-              cherished memories together...
+              Lorem ipsum odor amet, consectetuer adipiscing elit. Netus
```
Comment (Truniole) [inline index.html:45]: 👍

File: index.html [hunk 3]
```diff
@@ -41,20 +42,24 @@ <h1>Happy Hearts Roaming</h1>
...
+              libero finibus sodales in tristique. Velit et dignissim torquent
+              faucibus nunc. Egestas leo turpis feugiat volutpat felis...
             </p>
           </div>
           <a class="readmore-button" href="mystory.html">
             <button>READ MORE</button>
           </a>
         </div>
-        <img class="profile-pic" src="images/image3.jpg" alt="" />
+        <img class="profile-pic" src="https://via.placeholder.com/300" alt="" />
```
Comment (Truniole) [inline index.html:58]: This image is not working

File: myStory.html [hunk 4]
```diff
@@ -41,30 +42,34 @@ <h1>Happy Hearts Roaming</h1>
           <h1>My story</h1>
```
Comment (Truniole) [inline myStory.html:42]: The layout is broken after the css migration. 
Should this component be here?

**Review by Truniole — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (minna-le) [reply index.html:58]: I will replace it with actual images to better reflect how the design will appear

File: myStory.html (hunk 4 above)
Comment (minna-le) [reply myStory.html:42]: I haven't made any changes to the CSS for the 'My Story' page yet. The layout might look a bit different compared to the 'My Story' section on the landing page. I'll work on the CSS for the 'My Story' page once I've finished with the landing page.

**Review by Truniole — APPROVED** [review]

**Pull request merged**